packages/components/typography/src/typoTypes/body/Body4.tsx

```diff
@@ -0,0 +1,65 @@
+import type {
+  TypographyProps,
+  TypographyStyledProps,
+} from '../../typography.types';
+import { TypographyBodyStyled } from './typography.styles';
+import { deepMerge, useTheme } from '@douro-ui/react';
+import React from 'react';
+
+export const Body4 = ({
+  styled,
+  children,
+  bodyType = 'body4',
+  ...props
+}: TypographyProps): React.ReactNode => {
+  const theme = useTheme();
+
+  const getBody4DefaultThemeValues: TypographyStyledProps = {
+    fontSize: theme.typography.body.body4.mobile.fontSize,
+    lineHeight: theme.typography.body.body4.mobile.lineHeight,
+    fontWeight: theme.typography.body.body4.mobile.fontWeight,
+    color: theme.colors.brand.black,
+    fontFamily: theme.fontFamily.text,
+  };
+
+  const getResponsiveStyles = (): TypographyStyledProps => {
```

Comment:
This method could be a hook. The only difference between each component is the element (e.g. body4) and therefore could be passed as an argument.

Reply:
I have created the hook and updated the typography components' code